src/Breadcrumb/__stories__/__snapshots__/Breadcrumb.stories.storyshot

```diff
@@ -1,154 +1,142 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Storyshots Component API/Breadcrumb No Styles 1`] = `
-<div
-  dir="ltr"
+<ul
```

Comment:
All of these storyshot diffs are from using a different addon for LTR that's no longer captured in the snapshots